pageserver/src/repository.rs

```diff
@@ -69,7 +69,7 @@ pub enum RepositoryTimeline {
     /// Loaded into pageserver's memory and ready to be used.
     Local(Arc<dyn Timeline>),
     /// Timeline, found on the pageserver's remote storage, but not yet downloaded locally.
-    Remote(ZTimelineId),
+    Remote(ZTimelineId, Lsn),
```

Comment:
What do you think about using named struct with fields here? Or at least a comment what is the meaning of this Lsn